snapwell/snapwell_main.py

```diff
@@ -34,7 +34,7 @@ def percentage(value):
         raise argparse.ArgumentTypeError(f"Value must be float {value}")
     if 0.0 <= value <= 100.0:
         return value
-    raise argparse.ArgumentError(f"Value must be in range [0, 100] {value}")
+    raise argparse.ArgumentTypeError(f"Value must be in range [0, 100] {value}")
```

Comment:
This was intentional, but maybe it was incorrect :p

Reply:
ArgumentError requires you to give the value as the first parameter so that is why I changed it.